gapis/api/vulkan/transform_mapping_exporter.go

```diff
@@ -105,9 +109,55 @@ func (mappingTransform *mappingExporter) TransformCommand(ctx context.Context, i
 		mappingTransform.thread = inputCommands[0].Thread()
 	}
 
+	// This is a workaround for b/180532468.
+	// This does not solve the issue of mismapping due to duplication of Vulkan Handles.
+	// But it does reduces chance to hit the case by not mapping the framebuffer used in
+	// initialCmds.
+	if uint64(id.GetID()) >= mappingTransform.numOfInitialCmds {
+		for _, cmd := range inputCommands {
+			if queueSubmit, ok := cmd.(*VkQueueSubmit); ok {
+				if err := mappingTransform.recordFramebuffersInSubmittedRenderPasses(ctx, queueSubmit, inputState); err != nil {
+					return nil, err
+				}
+			}
+		}
+	}
+
 	return inputCommands, nil
 }
 
+// Record the framebuffers used in the renderpasses that actually submitted to GPU.
```

Comment:
Comment style: https://github.com/golang/go/wiki/CodeReviewComments#doc-comments

I'd also add the detail on what it populates, with the boiler plate of the function it takes a bit of effort to figure out what the state change on the object actually is.

// recordFramebuffersInSubmittedRenderPasses marks framebuffers that are submitted to the GPU in `mappingExporter.usedFrameBuffers`.